Sources/PackageType/c99name().swift

```diff
@@ -178,9 +178,9 @@ public func c99name(name: String) throws  -> String {
             0x4E00...0x9FA5,
             // Hangul,
             0xAC00...0xD7A3:
-            return true
+            return $0
         default:
-            return false
+            return "_"
         }
     }
```

Comment:
I think there is another change needed here, there is a loop below this one that deletes leading characters I believe.